Cargo.toml

```diff
@@ -13,8 +13,13 @@ members = [
     "testool"
 ]
 
-[patch.crates-io]
-halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2.git", tag = "v2023_02_02" } 
+# Uncomment after the next halo2 release
+# [patch.crates-io]
+# halo2_proofs = { git = "https://github.com/privacy-scaling-explorations/halo2.git", tag = "v2023_04_20" }
+
+# Remove after the next halo2 release
+[patch."https://github.com/privacy-scaling-explorations/halo2.git"]
+halo2_proofs = { git = "https://github.com/appliedzkp/halo2.git", rev = "be955686f86eb618f55d2320c0e042485b313d22" }
```

Comment:
Should we just override `v2023_04_20` with current latest halo2, or should we create a new release (I'd prefer the former for easier management for us)?